.teamcity/Gradle_Check/configurations/StagePasses.kt

```diff
@@ -67,6 +67,9 @@ class StagePasses(model: CIBuildModel, stage: Stage, prevStage: Stage?) : BaseGr
             tasks = "createBuildReceipt"
             gradleParams = "-PtimestampedVersion -Djava7Home=%linux.jdk.for.gradle.compile% -Djava7.home=%linux.jdk.for.gradle.compile% --daemon"
             useGradleWrapper = true
+            params {
+                param("env.JAVA_HOME", "%linux.java8.oracle.64bit%")
```

Comment:
Why don't you set top level params as here?
https://github.com/gradle/gradle/blob/4207cfec99f53cc7ff0ad8e1ee38f644cf149a5c/.teamcity/Gradle_Check/configurations/FunctionalTest.kt#L21-L27

Reply:
Fixed